Dockerfile

```diff
@@ -24,7 +24,10 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 # Copy the artifacts from `yarn build`
 COPY --from=builder /app/dist /action-release/dist/
 RUN chmod +x /action-release/dist/index.js
-ENV SENTRY_BINARY_PATH=/action-release/node_modules/.bin/sentry-cli
+
+# move the sentry-cli binary to where the entrypoint expects it
+RUN mv /action-release/node_modules/@sentry/cli/sentry-cli /action-release/sentry-cli
+RUN chmod +x /action-release/sentry-cli
```

Comment:
this isn't right -- something should be adjusted to find it in the original place